test/response_cache_handler_test.rb

```diff
@@ -24,6 +24,7 @@ def handler
       serve_unversioned: controller.send(:serve_unversioned_cacheable_entry?),
       cache_age_tolerance: controller.send(:cache_age_tolerance_in_seconds),
       headers: controller.response.headers,
+      controller: controller,
```

Comment:
Controller shouldn't be passed in since this gem is used in vanilla rack applications

Reply:
ah my bad, I didn't see that. Will open a new PR